CRVResponse/src/CrvCoincidenceFinder_module.cc

```diff
@@ -63,6 +63,15 @@ namespace mu2e
       fhicl::Atom<int> bigClusterThreshold{Name("bigClusterThreshold"), Comment("no coincidence check for clusters with a number of hits above this threshold")};
     };
 
+    // a vector that checks for duplicates
+    template <typename T> class uniqueVector : public std::vector<T> {
```

Comment:
It's always a little bit scary inheriting from an std:: container type.  They do not have virtual d'tors so there are use cases that will slice the object.

Another way to do this might be to use std::set but do define it with a user supplied sort function.  @ehrlich-uva : is there a sort ordering that will accomplish what we need?

Reply:
Yes, I can try to come up with a ordering function and add it to the set definition.